cli/test/e2e/package_repository_test.go

```diff
@@ -127,4 +127,21 @@ func TestPackageRepository(t *testing.T) {
 		require.Exactly(t, expectedOutputRows, output.Tables[0].Rows)
 	})
 
+	logger.Section("creating a repository in a new namespace that doesn't exist", func() {
```

Comment:
Just one change, can we add deletion of these two namespaces to the [cleanup function](https://github.com/carvel-dev/kapp-controller/pull/1113/files#diff-09f0c0ae3712103375872b02f78bf3bc30f715f8386e471d6235b5cf54d3e349R25)?
Maybe move the variables storing the names as well so that we can do that, something like `existingRepoNamesapce` and `newRepoNamespace` maybe.

Reply:
I'm trying doing this, but the tests are failing. I couldn't find a way to run the tests locally, so the troubleshooting is a bit slow.

Reply:
What do you think about this solution? https://github.com/carvel-dev/kapp-controller/pull/1113/commits/82dac38b2d06fbb4fadca77bb169dfa60a992440

Reply:
Sorry! Were we already deleting the namespaces I the Cleanup function? It looks like we are removing those bits in this commit. If that is the case we do not need this change

Reply:
From what I understood, the cleanup function is removing the package and related namespace in all test cases except the 2 new tests added because I use different namespaces. So I added the explicit cleanup for those 2 new test cases to delete both package and namespace since they are not caught by the cleanup function.

Reply:
(To put it concisely, I think the RemoveClusterResource call for these two namespaces should be within the cleanup function)